### DISCUSSION TITLE
How to make this flexbox layout ?

### DISCUSSION THREAD

**Discussion by davisse** [quasarframework/quasar#12359] [General - Components / Directives / etc]
Hello i would like to prepare my q-page for upcoming components and i'm learning flexbox as the same time.
i am working inside my Index.vue and i have a layout made with the layout builder. i would like to have 3 rows with different heights in it. 
![image](https://user-images.githubusercontent.com/16917861/152642639-2ebe8a8e-0360-4aa7-b1ee-f2ec997cb3b7.png)
I would like to have 2 rows of equal height and one last row using the remaining height available in the q-page. how can i achieve that please? because in the examples here https://quasar.dev/layout/grid/column#setting-one-row-height you need to manually setup the height in pixel and i dont want to have that ! 
thank you very much
David

**Comment by pdanpdan** [comment]
What is the rule for the 2 equal rows?

https://codepen.io/pdanpdan/pen/oNoYbLL?editors=1010
